app/src/main/java/com/fabulouszanna/pokedex/PokemonApp.kt

```diff
@@ -2,13 +2,13 @@ package com.fabulouszanna.pokedex
 
 import android.app.Application
 import com.fabulouszanna.pokedex.repo.MainDatabase
-import com.fabulouszanna.pokedex.repo.pokemon.SinglePokemonViewModel
+import com.fabulouszanna.pokedex.ui.pokemonlist.SinglePokemonViewModel
```

Comment:
AFAIU, this is for displaying details. Thus this model goes into `ui/pokemondetails`

Reply:
Addressed in next commit